utils/clean.py

```diff
@@ -97,5 +106,210 @@ def clean_state(self) -> (pd.DataFrame, pd.DataFrame, pd.DataFrame):
         Returns: use preprocess, clean, standardize, and create_tables methods
         to output (individuals_table, organizations_table, transactions_table)
         as defined in database schema
+
+        Inputs:
+            filepaths_list: list of absolute filepaths to relevant state data.
+                required naming conventions, order, and extensions
+                defined per state.
+
+        Returns: cleans the state and returns the standardized Inidividuals,
+        Organizations, and Transactions tables in a tuple
         """
+
+        # initial_dataframes = self.preprocess(filepaths_list)
+
+        # should run create tables, which runs through the functions above
+        # to preprocess, clean, standardizes and create the following tables
+        # (invividuals_table, organizations_table, transactions_table)
+
         pass
+
+
+class ArizonaCleaner(StateCleaner):
+    """This class is based on the StateCleaner abstract class,
+    and cleans Arizona data"""
+
+    def preprocess(filepaths: list[str]) -> list[pd.DataFrame]:
+        """Turns filepaths into dataframes
+
+        The input must be a list of valid filepaths which lead
+        to pandas dataframes. Typically, these should be just two
+        files: a transactions file and a details file, as
+        harvested by az_curl_crawler. If these conditions are not
+        met, the rest of the pipeline will not work
+
+        args: list of two filepaths for dataframes,
+        transactions and details, in that order
+
+        returns: a list of two dataframes, transactions and details,
```

Comment:
We'll discuss this on Friday; I can restructure it to fit like this, but three 3-table split would be artificial, not a natural consequence of the data scraped from the database. I worry it would be more confusing for later users